nerfstudio/viewer/server/render_state_machine.py

```diff
@@ -127,7 +128,13 @@ def _render_img(self, cam_msg: CameraMessage):
         assert camera is not None, "render called before viewer connected"
 
         with self.viewer.train_lock if self.viewer.train_lock is not None else contextlib.nullcontext():
-            camera_ray_bundle = camera.generate_rays(camera_indices=0, aabb_box=self.viewer.get_model().render_aabb)
+            # TODO jake-austin: Make this check whether the model inherits from a camera based model or a ray based model
+            # TODO Zhuoyang: First made some dummy judgements, need to be fixed later
+            isGaussianSplattingModel = isinstance(self.viewer.get_model(), GaussianSplattingModel)
+            if isGaussianSplattingModel:
+                pass
```

Comment:
```suggestion
                # TODO fix me before ship
                camera_ray_bundle = camera.generate_rays(camera_indices=0, aabb_box=self.viewer.get_model().render_aabb)
```

Reply:
Oops yes I only tested in the viewer_beta, since the old one will be deprecated soon. thanks for these fixes.

Reply:
tanku!  sure i mean like maybe in nerfstudio RayBundle should just also include the camera the rays came from haha